<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emulator</class>
 <widget class="QDialog" name="Emulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>RealTime</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>ADC</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>DAC</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3"/>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>TimeRef</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>ADC-V</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>R2, MOm</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6" colspan="2">
    <widget class="QLineEdit" name="lineEdit_4"/>
   </item>
   <item row="1" column="8">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Timer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5" rowspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>R3,GOm</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6" rowspan="2" colspan="2">
    <widget class="QLineEdit" name="lineEdit_5"/>
   </item>
   <item row="3" column="8" rowspan="2">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>U-offset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_8"/>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>C2, nF</string>
     </property>
    </widget>
   </item>
   <item row="5" column="6" colspan="2">
    <widget class="QLineEdit" name="lineEdit_6"/>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>R1</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>I</string>
     </property>
    </widget>
   </item>
   <item row="6" column="6" colspan="2">
    <widget class="QLineEdit" name="lineEdit_7"/>
   </item>
   <item row="7" column="0" colspan="7">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>СТАТУС</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="8" column="4" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="8" column="7" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>recieveADC</string>
     </property>
    </widget>
   </item>
   <item row="9" column="7" colspan="2">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>recieveCOM</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
